README.md

```diff
@@ -0,0 +1,39 @@
+Staff Fee Privilege API Load Testing
+---
+
+The purpose of this repo is to load balance the [Staff Fee Privilege API](https://github.com/osu-mist/staff-fee-privilege-api) using [Locust](https://github.com/locustio/locust).
+
+## Usage
+
+  1. Install all dependencies via pip:
+
+      ```
+      $ pip install -r requirements.txt
+      ```
+
+  2. Copy [config-example.yaml](config-example.yaml) as `config.yaml`. Modify as necessary, being careful to avoid committing sensitive data.
+
+  3. Run API locally (e.g. https://localhost:8080) then start locust with Locust file:
+
+      ```
+      $ locust -f locustfile.py --host=https://localhost:8080
+      ```
+
+      Once you’ve started Locust, you should be able to open up a browser and point it to http://localhost:8089 to access Locust’s web interface.
+
+## Report
```

Comment:
How long did it take to make the requests in each test (ex. 2029 in 10 seconds? 20 seconds?)? What was the average response time per API call for the various tests?